README.md

```diff
@@ -305,6 +306,50 @@ The default `interval` is `50ms`. However it will run your callback immediately
 on the next tick of the event loop (in a `setTimeout`) before starting the
 intervals.
 
+### `waitForElements`
+
+Defined as:
+
+```typescript
+function waitForElements<T>(
```

Comment:
Normally I think people will be looking for a single element. Should this be called `waitForElement`?

In addition, we should show an example (in the FAQ?) of getting an array of elements:

```javascript
const elements = waitForElement(() => [
  getByText('username'),
  getByText('password')
])
```